app/controllers/concerns/idv/ab_test_analytics_concern.rb

```diff
@@ -0,0 +1,26 @@
+module Idv
+  module AbTestAnalyticsConcern
+    def ab_test_analytics_args
```

Comment:
High level question, why did we pick `_args` for this? When I read `_args` I think that these items are going to be handed into something else to make a decision? But it looks like this is returning the buckets the user was assigned to instead, so we can pass that to our logs?

What if we named these:
```suggestion
    def ab_test_analytics_buckets
```

Reply:
They're arguments to analytics, so analytics_args. Your suggestion makes sense, will change.